public/lib/dragAndDroplayercontrol/DndLayerControl.js

```diff
@@ -589,6 +589,9 @@ function getExtendedMapControl() {
         if (spatialPathSource.geometrytype && spatialPathSource.geometrytype.includes('Polygon')) {
           geometryType = 'polygon';
         }
+        if (spatialPathSource.geometrytype && spatialPathSource.geometrytype.includes('Line')) {
```

Comment:
I would tidy it up by doing something like this at the top of your if statement:
const geometryType = _.get(spatialPathSource, 'geometrytype');

Then change all of them to: 
```suggestion
        
        if (geometryType === 'MultiLineString' || geometryType === 'LineString') {
```

Reply:
Done.